src/Symfony/Component/HttpKernel/EventListener/AbstractTestSessionListener.php

```diff
@@ -31,8 +31,8 @@
  */
 abstract class AbstractTestSessionListener implements EventSubscriberInterface
 {
-    private $sessionId;
-    private $sessionOptions;
+    private ?string $sessionId = null;
```

Comment:
could me rewritten to be uninitialized

Reply:
> null isn't meant to be a legal sessionid

No, but not having a session ID is a legal state of the listener and not an indicator for a forgotten initialization. I'm also in favor of not making properties nullable if I don't have to, but it having a nullable session ID looks okay to me here.